src/Symfony/Component/Mailer/Transport/Smtp/EsmtpTransport.php

```diff
@@ -106,7 +106,7 @@ protected function doHeloCommand(): void
 
         /** @var SocketStream $stream */
         $stream = $this->getStream();
-        if (!$stream->isTLS() && \defined('OPENSSL_VERSION_NUMBER') && \array_key_exists('STARTTLS', $capabilities)) {
+        if ($stream->isTLS() && \defined('OPENSSL_VERSION_NUMBER') && \array_key_exists('STARTTLS', $capabilities)) {
```

Comment:
That change is wrong. If the SMTP server tells us that it supports STARTTLS, then we are using it. I closed a very similar change recently.

Reply:
indeed... sounds like I've been trapped in the same issue than @lyrixx => https://github.com/maildev/maildev/issues/274